### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by martonpe**
### Added
 
* --data option for extra data to be added to the message payload

### Changed
 
* option --broker renamed to --broker-list and accpeting a comma separated list of brokers

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,12 +3,16 @@
...
-## [Unreleased]
+## 0.2.0 (2015-06-12)
 
 ### Added
 
+  * --data option for extra data to be added to the message payload
+
 ### Changed
 
+  * option --broker renamed to --broker-list and accpeting a comma separated list of brokers
```
Comment (rgabo) [inline CHANGELOG.md]: small typo and spelling is "comma-separated"

you should also use statements in simple tense, as in "option --broker renamed to --broker-list and accepts comma-separated list of brokers"

File: bin/avro2kafka [hunk 2]
```diff
@@ -10,8 +10,8 @@ option_parser = OptionParser.new do |opts|
   opts.banner = "Version #{Avro2Kafka::VERSION} of Avro2Kafka\n" \
     "Usage: #{File.basename(__FILE__)} [options] [file]"
 
-  opts.on('-b', '--broker BROKER', 'The Kafka broker, eg. localhost:9092. This value is required.') do |broker|
-    options[:broker] = broker
+  opts.on('-b', '--broker-list BROKER', 'The Kafka broker list. Format: host:port. This value is required.') do |broker_list|
+    options[:broker_list] = broker_list
```
Comment (rgabo) [inline bin/avro2kafka]: Parsing and validation of the argument `broker_list` should be done here and only an array of brokers passed as `options`.

Comment (rgabo) [inline bin/avro2kafka]: Well, maybe not. I take it back, `Avro2Kafka` is responsible for the CLI and it makes it easier to test.

**Comment by rgabo** [comment]
I realize it was time sensitive, but the [gem](https://rubygems.org/gems/avro2kafka/versions/0.2.0) should not be released/pushed until the actual release is made here on GitHub :wink:

We should have a way to package a development/latest version of the gem up in Docker. A private gem registry is an option but even easier is if we write a Dockerfile that builds the latest version of the gem itself.

**Comment by rgabo** [comment]
Depends on #5

**Comment by rgabo** [comment]
@martonpe great release! TravisCI should be set up to run the tests on many Ruby versions. I couldn't easily determine what our Ruby version requirements were. It's definitely 2.x as we use keyword parameters,but I don't know if it works with 2.0 or only 2.1.

**Comment by martonpe** [comment]
@rgabo It should work with 2.0, couldn't find any dependency on 2.1.

**Comment by rgabo** [comment]
@martonpe let's have TravisCI confirm it and also test with other Rubies.

**Pull request merged**